board/stm32h7/clock.h

```diff
@@ -17,6 +17,13 @@ APB4 per: 60MHz
 PCLK1: 60MHz (for USART2,3,4,5,7,8)
 */
 
+// TODO: find a better way to distinguish between H723 and H725
+// The package will do for now, since we have only used TFBGA100 for H723
+bool is_h723(void) {
+  uint8_t package = SYSCFG->PKGR & 0xFU;
```

Comment:
is `SYSCFG` enabled at this point? I don't think this is expected to be valid until the peripheral is enabled.

in the bootstub:
https://github.com/commaai/panda/blob/f6375848ca393a9483921665b6a2d131d7ec9b20/board/bootstub.c#L37-L43

in the app (which the CR3 behavior makes irrelevant):
https://github.com/commaai/panda/blob/f6375848ca393a9483921665b6a2d131d7ec9b20/board/main.c#L300-L303

Reply:
I assumed this wasn't clock gated, but I might be wrong (will check the reference manual).

It does seem to read out correctly at this point after a power cycle, but good to make sure.

Reply:
enabled it before reading out. APB4 should have some frequency on it already from boot